group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTestContext.java

```diff
@@ -901,7 +901,7 @@ public RebalanceResult staticMembersJoinAndRebalance(
     public PendingMemberGroupResult setupGroupWithPendingMember(ClassicGroup group) throws Exception {
         // Add the first member
         JoinGroupRequestData joinRequest = new JoinGroupRequestBuilder()
-            .withGroupId("group-id")
+            .withGroupId(group.groupId())
```

Comment:
(this is unrelated to this PR)

The caller `GroupMetadataManagerTest` has similar issue (https://github.com/apache/kafka/blob/trunk/group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java#L7631). Should it get fixed also?